src/features/courses/course-route.ts

```diff
@@ -1041,7 +1065,16 @@ router.get('/:id',
         try {
             const params = req.params as GetCourseRequest.params;
             const course = await courseController.getCourseById(params.id);
-            next(httpResponse.Ok('Fetched successfully', course));
+            const university = await course.getUniversity({
+                where: {
+                    active: true,
+                }
+            });
+            const canAskForHelp = university?.universityName === 'CityTech' ?? false;
```

Comment:
This is great for now, though I wonder if there is some unique identifier that belongs to the university rather than our application that we can use

Once we start showing the university name in the application we might update it and then this will break

Reply:
(There's nothing to do here)

Reply:
It would have to be another field in the db, rather than a natural identifier, for this to be a truly OSS solution.